aws/b2b_personalization/modules/es/outputs.tf

```
@@ -0,0 +1,14 @@
+output "elasticsearch_username_ssm_parameter_name" {
+  description = "Name of the Elasticsearch username SSM parameter"
+  value       = aws_ssm_parameter.elasticsearch_username.name
+}
```

I have to admit that for values ​​that aren't secrets, it's easier to return the value instead of the name
```suggestion
output "elasticsearch_username" {
  description = aws_ssm_parameter.elasticsearch_username.description
  value       = aws_ssm_parameter.elasticsearch_username.value
}
```